src/Layout/QueryParamContext.tsx

```diff
@@ -261,7 +259,11 @@ export default function QueryParamProvider({ children }: any) {
       const config: any = await apiRequest(requestParams);
       if (config.data) {
         if (!router.query.country) {
-          setcountry(config.data.country);
+          if (config.data.country == "XX" || config.data.country == "T1") {
```

Comment:
Can we check whether the country is available in the array and if it is present we set the country or we set it to DE

```
if (country) { 
	   const found = countriesData.some( 
	     (arrayCountry) => arrayCountry.countryCode?.toUpperCase() === country?.toUpperCase() 
	   ); 
	   if (found) { 
	    setcountry(country);
	   } else{
	    setcountry('DE');
	   }
	 } 
```

Reply:
sure